Xamarin.Forms.Xaml/MarkupExtensions/BindingExtension.cs

```diff
@@ -18,6 +18,13 @@ public sealed class BindingExtension : IMarkupExtension<BindingBase>
 		public object FallbackValue { get; set; }
 		public TypedBindingBase TypedBinding { get; set; }
 
+		// Here to maintain parity with other XAML flavors.
+		public RelativeBindingSource RelativeSource
+		{
+			get => this.Source as RelativeBindingSource;
+			set => this.Source = value;
+		}
+
```

Comment:
please remove this

Reply:
Ok you're the boss, boss, But I just want to make sure we're 100% clear that this would deviate from the WPF/UWP syntax. If it's the redundancy that bothers you, as an XForms consumer I'd rather require the use of `RelativeSource=` than make `Source=` and `RelativeSource=` interchangeable, just to minimize differences between the platforms. Please confirm?